autobump-maven/src/main/java/com/github/autobump/maven/model/MavenDependencyResolver.java

```diff
@@ -22,6 +33,36 @@ public MavenDependencyResolver() {
         Model model = mavenModelAnalyser.getModel(workspace);
         Set<Dependency> dependencies = getDependencies(model);
         dependencies.addAll(getPlugins(model));
+        var modules = model.getModules();
```

Comment:
it might be cleaner if you pass the this value to the method resolveModules and let it return a set of all the dependencies in the found modules